packages/components/src/FormField/components/ClearAction.css

```diff
@@ -0,0 +1,25 @@
+.clearContainer {
+  display: flex;
+  align-items: center;
+  padding-left: var(--space-small);
+}
+
+.clearInput {
+  display: flex;
+  position: relative;
+  right: var(--space-base);
+  z-index: var(--elevation-tooltip);
+  width: var(--space-large);
+  height: var(--space-large);
+  padding: 0;
+  border: none;
+  border-radius: var(--radius-circle);
+  background-color: var(--color-surface--background);
+  cursor: pointer;
+  justify-content: center;
+  align-items: center;
+}
+
+.addedMargin {
```

Comment:
What do we need so much margin for?

Reply:
oh yeah, I added it before because I thought it would be the clear button on the left side of the timer/number picker. but it's a good catch, it's not being used anymore.